### PR TITLE
feat(ui): support custom properties for glossary, database & databaseSchema page

### LINKED ISSUE #12898: customProperties for glossaryTerm
**Is your feature request related to a problem? Please describe.**
To be able to add custom properties (attributes) to glossaryTerm.  Currently only available for tables and so on. https://docs.open-metadata.org/v1.0.0/how-to-guides/how-to-add-custom-property-to-an-entity

**Describe the solution you'd like**
glossaryTerm is entity as others and would benefit of this geature.

add attribute, with different datatypes: boolean, reference value list, numeric, string
for the attribute to be visible on the UI

**Describe alternatives you've considered**
no other options

**Additional context**
UI says Custom Attribute while docs say custom property or extensions

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes:

Fixes #12898, #11287

I worked on below tasks:

- [x] Support custom properties for Glossary Terms

https://github.com/open-metadata/OpenMetadata/assets/12962843/f515243a-406f-48e5-a170-65f3f8fac4ad

Screenshot 2023-09-16 at 9 09 32 PM

- [x] Support custom properties for Database details page

https://github.com/open-metadata/OpenMetadata/assets/12962843/c0068437-422d-4043-ae03-fd7c01e9c188

- [x] Support custom properties for Database schema page

https://github.com/open-metadata/OpenMetadata/assets/12962843/27845fc7-4954-4904-b3e4-61aea039c5c9

- [ ] Need to replace database icon

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by aniketkatkar97 — COMMENTED** [review]
Left one comment

File: openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaVersionPage/DatabaseSchemaVersionPage.tsx [hunk 1]
```diff
@@ -290,7 +331,9 @@ function DatabaseSchemaVersionPage() {
                 <Tabs
                   className="entity-details-page-tabs"
                   data-testid="tabs"
+                  defaultActiveKey={tab ?? EntityTabs.SCHEMA}
```
[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaVersionPage/DatabaseSchemaVersionPage.tsx]: Updated thanks

**Review by aniketkatkar97 — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryHeader/GlossaryHeader.component.tsx [hunk 2]
```diff
@@ -143,9 +142,7 @@ const GlossaryHeader = ({
   const handleVersionClick = async () => {
     let path: string;
     if (isVersionView) {
-      path = isGlossary
-        ? getGlossaryPath(latestGlossaryData?.fullyQualifiedName)
-        : getGlossaryTermsPath(latestGlossaryData?.fullyQualifiedName ?? '');
+      path = getGlossaryPath(latestGlossaryData?.fullyQualifiedName);
```
Comment (karanh37) [inline openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryHeader/GlossaryHeader.component.tsx:145]: why did we remove glossary terms path here ?

File: openmetadata-ui/src/main/resources/ui/src/utils/CustomProperties/CustomProperty.utils.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import { getGlossaryTermByFQN } from 'rest/glossaryAPI';
+import { getMlModelByFQN } from 'rest/mlModelAPI';
+import { getPipelineByFqn } from 'rest/pipelineAPI';
+import { getSearchIndexDetailsByFQN } from 'rest/SearchIndexAPI';
+import { getContainerByFQN } from 'rest/storageAPI';
+import { getStoredProceduresDetailsByFQN } from 'rest/storedProceduresAPI';
+import { getTableDetailsByFQN } from 'rest/tableAPI';
+import { getTopicByFqn } from 'rest/topicsAPI';
+
+export const getEntityExtentionDetailsFromEntityType = <
```
Comment (karanh37) [inline openmetadata-ui/src/main/resources/ui/src/utils/CustomProperties/CustomProperty.utils.ts:29]: Can we utilise same method which we have used under AssetsUtils `getAPIfromSource`.

**Review by karanh37 — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryHeader/GlossaryHeader.component.tsx (hunk 2 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryHeader/GlossaryHeader.component.tsx:145]: we aren't using those, We just determine glossary or glossaryTerm from the split function from FQN class

**Review by karanh37 — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**